clients/e-shop-demo/e-shop-frontend/src/components/authorize-credential/authorize-credential.component.tsx

```diff
@@ -0,0 +1,35 @@
+import { useContext, useEffect, useState } from 'react';
+import { UserContext } from '../../contexts/user.provider';
+import { Input, SmallButton } from '../../global.styles';
+import { authenticate } from '../../services/authentication.service';
+import MessageBox from '../message-box/message-box.component';
+
+const AuthorizeCredential = () => {
+	const [error, setError] = useState(false);
+	const [signedNonce, setSignedNonce] = useState<string>('');
+	const { userIdentityId, setAuthenticated } = useContext(UserContext);
+
+	const login = async () => {
+		setError(false);
```

Comment:
this shouldn't be necessary

Reply:
Is necessary for resetting the error onLogin when an error appeard before.